.github/workflows/ci.yml

```diff
@@ -16,8 +16,8 @@ env:
   # maven.wagon.rto is in millis, defaults to 30m
   MAVEN_OPTS: "-Xmx512M -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
   MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
-  MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -DskipTests -Dair.check.skip-all"
-  MAVEN_TEST: "-B --strict-checksums -Dair.check.skip-all --fail-at-end"
+  MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -Dair.check.skip-all -P ci -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }}"
```

Comment:
This assumes the workflow will only ever be triggerred via pull requests (the reference branch being passed in). What happens when it's a direct push to `master` (for any reason).

Reply:
The `ci` profile already exists in server-rpm module and it runs the rpm integration tests - should it run as part of MAVEN_FAST_INSTALL?

Reply:
GIB is disabled on master (`<disableIfBranchMatches>master</disableIfBranchMatches>`), so this parameter won't be effective.

I reused the `ci` profile instead of creating a new one but I checked that `trino-server-rpm` is excluded everywhere `MAVEN_FAST_INSTALL` is used. If you think I should add a new profile, do you have any suggestions for the name?

Reply:
I believe `gib` is a good profile name - better to be explicit instead of depending on the fact that `ci` is underused today.

Reply:
Done!